code/modules/power/supermatter/supermatter.dm

```diff
@@ -945,57 +949,57 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 		default_unfasten_wrench(user, tool, time = 20)
 	return TRUE
 
-/obj/machinery/power/supermatter_crystal/Bumped(atom/movable/AM)
-	if(isliving(AM))
-		AM.visible_message("<span class='danger'>\The [AM] slams into \the [src] inducing a resonance... [AM.p_their()] body starts to glow and burst into flames before flashing into dust!</span>",\
+/obj/machinery/power/supermatter_crystal/Bumped(atom/movable/object)
```

Comment:
Which object? Again, context is important--`hit_object` for example is much better.